systemtests/test_env.py

```diff
@@ -0,0 +1,136 @@
+#!/usr/bin/env python
+
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+import os
+
+import click
+
+from systemtests.bin.download_sym_files import download_sym_files
+from systemtests.bin.list_firefox_symbols_zips import list_firefox_symbols_zips
+from systemtests.bin.upload_symbols import upload_symbols
+from systemtests.bin.upload_symbols_by_download import upload_symbols_by_download
+
+ZIPS_DIR = "./data/zip-files/"
+USAGE = "Usage: test_env.py [local|stage|prod]"
+
+
+@click.command()
+@click.argument("env")
+def test_env(env):
+    """
+    Runs through downloading and uploading tests.
+
+    To use:
+    1. run "make shell" to get a shell in the container
+    2. then do "cd systemtests"
+    3. run "./test_env.py [ENV]"
+
+    To set auth tokens, add these to your .env file:
+    * LOCAL_AUTH_TOKEN
+    * STAGE_AUTH_TOKEN
+    * PROD_AUTH_TOKEN
+    """
+    destructive_tests = 0
+    bad_token_test = 0
+    tecken_host = ""
+    auth_token = ""
+
+    try:
+        if env == "local":
+            destructive_tests = 1
+            bad_token_test = 1
+            tecken_host = "http://web:8000/"
+            auth_token = f"{os.environ['LOCAL_AUTH_TOKEN']}"
+        elif env == "stage":
+            destructive_tests = 1
+            bad_token_test = 1
+            tecken_host = "https://symbols.stage.mozaws.net/"
+            auth_token = f"{os.environ['STAGE_AUTH_TOKEN']}"
+        elif env == "prod":
+            print("Running tests in prod--skipping destructive tests!")
+            tecken_host = "https://symbols.mozilla.org/"
+            auth_token = f"{os.environ['PROD_AUTH_TOKEN']}"
+        else:
+            print(f"{USAGE}")
```

Comment:
Interestingly, if I make this change, the `test_env.py` script doesn't exit.

I added `sys.exit(1)` to the end of the `except` block, and that worked as expected. Is this what you had in mind?

With suggested change:
```
app@4a4523ea4f82:/app/systemtests$ ./test_env.py wow
Unexpected error: ENV must be one of 'local', 'stage', or 'prod'.
tecken_host: 

>>> SKIPPING DESTRUCTIVE TESTS



>>> DOWNLOAD TEST
NOTE: None of these should take over 1000ms to download.

Traceback (most recent call last):
  File "/app/systemtests/./test_env.py", line 142, in <module>
    test_env()
  File "/usr/local/lib/python3.11/site-packages/click/core.py", line 1157, in __call__
    return self.main(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/click/core.py", line 1078, in main
    rv = self.invoke(ctx)
         ^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/click/core.py", line 1434, in invoke
    return ctx.invoke(self.callback, **ctx.params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/click/core.py", line 783, in invoke
    return __callback(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/app/systemtests/./test_env.py", line 130, in test_env
    download_sym_files(
  File "/usr/local/lib/python3.11/site-packages/click/core.py", line 1157, in __call__
    return self.main(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/click/core.py", line 1078, in main
    rv = self.invoke(ctx)
         ^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/click/core.py", line 1434, in invoke
    return ctx.invoke(self.callback, **ctx.params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/click/core.py", line 783, in invoke
    return __callback(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/app/systemtests/bin/download_sym_files.py", line 70, in download_sym_files
    sym_filename, expected_status_code, bucket, _ = parts
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
ValueError: not enough values to unpack (expected 4, got 3)
```

With suggested change plus a call to `sys.exit(1)` in the `except` block:
```
app@4a4523ea4f82:/app/systemtests$ ./test_env.py wow
Unexpected error: ENV must be one of 'local', 'stage', or 'prod'.
```

Reply:
Edit: when combined with the diff in https://github.com/mozilla-services/tecken/pull/2885#discussion_r1491497400, I don't need the `sys.exit(1)` call after all.

Reply:
> I don't know why there's a try/except that prints an Unexpected error. I would remove that outer block.

There's a scenario where the auth token isn't set in the env where the outer `try` block catches a `KeyError` with a descriptive message.  If I remove it, we'll still get `KeyError: 'LOCAL_AUTH_TOKEN'` at the bottom of a messy trace. I'll change it to that, since it seems clear enough if that works for you?

Reply:
That works! :+1: